src/main/java/com/datapipe/jenkins/vault/credentials/common/AbstractVaultBaseStandardCredentials.java

```diff
@@ -23,7 +22,6 @@ public abstract class AbstractVaultBaseStandardCredentials extends BaseStandardC
     private String prefixPath;
     private String namespace;
     private Integer engineVersion;
-    private ItemGroup context;
```

Comment:
Are we sure this is required to be removed?

As this would allow lookup of credentials based on folder context, which users have reported as a bug previously.

Reply:
That was copied from #206 without considering the consequence and I haven't tested folders. Let me check this.

Reply:
@jamesrobson-secondmind much appreciated!
Absolute hero for testing all of this and carrying it over!

Reply:
Using folder level creds do not work correctly with this change.